aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -319,7 +319,6 @@ at::Tensor PackedConvWeightCudnn<kSpatialDim>::apply_impl(
     double output_scale,
     int64_t output_zero_point) {
   const int N = act.size(0);
-  const int D = kSpatialDim == 3 ? act.size(2) : 1;
   const int H = act.size(kSpatialDim);
   const int W = act.size(kSpatialDim + 1);
   const int M = orig_weight_.size(0); // output channels
```

Comment:
looks like we already have a variable here for output_channels, maybe we can rename this to make it clearer and use this one